server/Makefile

```
@@ -0,0 +1,15 @@
+start-redis:
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)